### PR TITLE
[master] Add language to Page

### LINKED ISSUE #95: [i18n] Add Page language

### PR DESCRIPTION
**Pull request by FranckKe**
- [x] Add language field to page
- [x] Add dropdown with available languages on page create/edit
- [x] Add language to seed
- [x] Only show pages in current language in menu, fallback to pages with default language if no pages are available in current language

Fixes #95

### CODE REVIEW

**Comment by mdouchement** [comment]
You can put simple data migration within a database migration like this:

```rb
class AddLanguageToPages < ActiveRecord::Migration[5.1]
  def change
    add_column :pages, :language, :string
    Page.update_all language: 'fr'
  end
end
```

Ensure that the rollback does not raise an error.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -12,7 +12,9 @@ def status
   private
 
   def fetch_pages
-    @pages = Page.all # For side menu
+    # List pages for side menu
+    @pages = Page.where(language: I18n.locale)
```
Comment (mdouchement) [inline app/controllers/application_controller.rb]: Keep in mind that an index on `language` would be useful for a large number of pages and languages.
IMO it is not needed for the current amount of pages.

File: app/models/concerns/page_manager.rb [hunk 2]
```diff
@@ -6,18 +6,37 @@ module PageManager
...
+          field :language
           field :content
         end
-
         edit do
           field :name
+          field :language, :enum do
+            enum do
+              [
+                %w[Français fr],
```
Comment (mdouchement) [inline app/models/concerns/page_manager.rb]: Is this array can be populated from the loaded locales?

Comment (FranckKe) [reply app/models/concerns/page_manager.rb]: Yeah, working on that actually

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -12,32 +12,42 @@ def status
...
     options.merge(locale: I18n.locale)
   end
 
   def languages
-    @languages ||= ActiveSupport::HashWithIndifferentAccess.new(
-      en: 'English',
-      es: 'Español',
-      fr: 'Français'
-    )
+    @languages ||= ActiveSupport::HashWithIndifferentAccess.new.tap do |langs|
```
Comment (mdouchement) [inline app/controllers/application_controller.rb]: @FranckKe I am wondering if this method can be totally static and put in an initializer?

**Pull request merged**